### DISCUSSION TITLE
Ignore version

### DISCUSSION THREAD

**Discussion by waghanza** [renovatebot/renovate#23857] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Having a ignore command on something like that to avoid extra PR

### What you tried so far.

Nothing

### Relevant debug logs

- I have enabled `renovatebot` on my repo
- This repo contains a lib X that use a deps Y
- A new release occurs on deps Y
- The current (major) of lib X can not use the new version of deps Y
- How can I ignore `renovatebot` update PR for deps Y until lib X is released ?

Deps Y is `jetty`
Lib X is `struts2`

Related to https://github.com/the-benchmarker/web-frameworks/pull/6584

**Comment by rarkins** [comment, marked as answer]
The problem here is that [org.eclipse.jetty:jetty-servlet](https://eclipse.org/jetty) and [org.eclipse.jetty:jetty-server](https://eclipse.dev/jetty) share a variable `jetty.version` but their latest major release is different.

Possible solutions which don't require Renovate coding:
- don't group them together if they aren't using synchronized releases
- configure separateMultipleMajor=true for these packages

The latter will mean that you have upgrades proposed for v10, v11 and v12 separately, any of which you can accept or "close to ignore". The close to ignore doesn't work because of the mixing of v11 and v12
